### PR TITLE
#6 do create admin field in customer table

### PR DESCRIPTION
**Pull request by AhmedHelalAhmed**
create admin field in customer table done.

### CODE REVIEW

File: app/models/Customer.php [hunk 1]
```diff
@@ -63,18 +69,19 @@ public function setPhone($phone)
         $this->phone = $phone;
     }
 
+
+    public function setIsadmin($flag)
+    {
+        $this->isadmin = $isadmin;
```
Comment (tramyardg) [inline app/models/Customer.php]: Is this line supposed to be `$this->isadmin = $flag;`

Comment (AhmedHelalAhmed) [reply app/models/Customer.php]: You are right

Comment (AhmedHelalAhmed) [reply app/models/Customer.php]: I will fix Seconds

**Comment by AhmedHelalAhmed** [comment]
Isadmin setter fixed

**Comment by tramyardg** [comment]
Thanks @AhmedHelalAhmed

**Pull request merged**